Makefile

```diff
@@ -30,7 +30,7 @@ $(unsupported):
 	fi
 
 opensuse/tumbleweed: setup = zypper --non-interactive install libglib-2_0-0
-manjarolinux/base: setup = pacman -Sy --noconfirm archlinux-keyring
+manjarolinux/base: setup = mv /etc/pacman.conf{.pacnew,} || true
```

Comment:
I don't understand what this does. Was that `pacman` command unnecessary? Where is this temp config file coming from?

Reply:
And we no longer need to install/update the `archlinux-keyring` package?